das/metrics.go

```diff
@@ -114,12 +114,18 @@ func (m *metrics) observeSample(ctx context.Context, h *header.ExtendedHeader, s
 	if m == nil {
 		return
 	}
+
 	m.sampleTime.Record(ctx, sampleTime.Seconds(),
 		attribute.Bool("failed", err != nil),
-		attribute.Int("header_width", len(h.DAH.RowsRoots)))
+		attribute.Int("header_width", len(h.DAH.RowsRoots)),
+		attribute.Int("header", int(h.RawHeader.Height)),
```

Comment:
If header high used as attribute, it will be evaluated as vector dimension in the prometheus. Header hight will have very high cardinality and will totally kill prometheus performance. Thats the primary reason I've not added it as an attribute.

Please consider using it as a metric value(Gauge) if it is a must for your observation or use an aggregated value for observation like `das_sampled_chain_head`. 
Please

Reply:
Also, @rootulp was advocating long ago to keep our metrics with low cardinality and not to overwhelm our incentivized testnet OTEL collector and Prometheus behind it.